src/Commands/RunScan.ts

```diff
@@ -211,3 +213,15 @@ export class RunScan implements CommandModule {
     }
   }
 }
+
+async function fetchTestTypes(baseURL: any): Promise<string[]> {
```

Comment:
The command is not the right place for such logic. Extract it to a separate class.